server/service/src/sync/translations/stock_line.rs

```diff
@@ -165,15 +179,21 @@ impl SyncTranslation for StockLineTranslation {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use repository::{mock::MockDataInserts, test_db::setup_all};
+    use repository::{
+        mock::MockDataInserts, test_db::setup_all, ChangelogFilter, ChangelogRepository,
+    };
+    use serde_json::json;
 
     #[actix_rt::test]
     async fn test_stock_line_translation() {
         use crate::sync::test::test_data::stock_line as test_data;
         let translator = StockLineTranslation {};
 
-        let (_, connection, _, _) =
-            setup_all("test_stock_line_translation", MockDataInserts::none()).await;
+        let (_, connection, _, _) = setup_all(
```

Comment:
Do we really need to add these ?

Reply:
No, great catch!

Reply:
eaccf46ed12bc74bf63195e74ca5115aabe7c60b